app/javascript/components/supplemental_application/SupplementalApplicationContainer.js

```diff
@@ -120,12 +108,7 @@ const StatusHistorySection = withContext(({ store, formIsValid }) => {
 })
 
 class SupplementalApplicationContainer extends React.Component {
-  constructor (props) {
-    super(props)
-    this.state = {
-      failed: false
-    }
-  }
+  state = { failed: false }
```

Comment:
Saw a few refactors in here that I took advantage of for removing constructors and direct returns on arrow function components